x-govuk/components/masthead/README.md

```diff
@@ -85,3 +86,10 @@ If you’re using Nunjucks macros in production with `html` options, or ones end
 | :--- | :--- | :---------- |
 | **src** | string | URL of image displayed on the right of the masthead on desktop layouts. |
 | **alt** | array | Alternative text for image. |
+
+### Options for gridClass
+
+| Name | Type | Description |
+| :--- | :--- | :---------- |
+| **start** | string | The [govuk grid class](https://design-system.service.gov.uk/styles/layout/#grid-system) applied to the container for the title, description and button elements. Defaults to `govuk-grid-column-two-thirds-from-desktop`. |
+| **end** | array | The [govuk grid class](https://design-system.service.gov.uk/styles/layout/#grid-system) applied to the container for the image elements. Defaults to `govuk-grid-column-one-third-from-desktop` if an image is supplied. |
```

Comment:
How about `params.title.gridClasses` and `params.image.gridClasses`, so that it's clear what the classes relate to, and also that you can pass in a string with multiple classes.

An alternative might be `wrapperClasses`, as the classes don't _have_ to be grid related

Reply:
Agree - this is a more holistic fix. Feel free to close this PR @paulrobertlloyd